spec/jobs/send_webhook_job_spec.rb

```diff
@@ -338,6 +338,28 @@
     end
   end
 
+  context 'when webhook_type is alert.subscription_to_be_terminated' do
+    let(:webhook_service) { instance_double(Webhooks::Subscriptions::TerminatingSoonService) }
+    let(:subscription) { create(:subscription) }
+
+    before do
+      allow(Webhooks::Subscriptions::TerminatingSoonService).to receive(:new)
+       .with(object: subscription, options: {}, webhook_id: nil)
+       .and_return(webhook_service)
```

Comment:
Layout/MultilineMethodCallIndentation: Use 2 (not 1) spaces for indenting an expression spanning multiple lines.